### PR TITLE
Refresh the Galaxy docs

### PR DESCRIPTION
**Pull request by chouseknecht**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

galaxy.rst
##### ANSIBLE VERSION

It was pointed out to me on [Galaxy issue #202](https://github.com/ansible/galaxy-issues/issues/202) that this doc was quite outdated and needed some attention.

### CODE REVIEW

**Comment by chouseknecht** [comment]
@bcoca can you please give this a once-over when you have a minute. I want to make sure my re-write is technically accurate. I'm sure I missed something.

Thanks!

File: docsite/rst/galaxy.rst [hunk 1]
```diff
@@ -1,105 +1,104 @@
...
+::
+
+    $ ansible-galaxy install username.role_name
 
 roles_path
 ==========
 
-You can specify a particular directory where you want the downloaded roles to be placed::
+Be aware that Ansible downloads roles to *ANSIBLE_ROLES_PATH*, which defaults to */etc/ansible/roles*. You can override this by setting the environment 
+variable in your session, defining *roles_path* in an *ansible.cfg* file, or by using the *--roles-path* option.
```
Comment (bcoca) [inline docsite/rst/galaxy.rst]: roles path can be a series of directories i.e. `/etc/ansible/roles:~/.ansible/roles`, the default is correct but it should install to the 'first writable path configured'

File: docsite/rst/galaxy.rst [hunk 2]
```diff
@@ -1,105 +1,104 @@
...
 
-To install multiple roles, the ansible-galaxy CLI can be fed a requirements file.  All versions of ansible allow the following syntax for installing roles from the Ansible Galaxy website::
+    $ ansible-galaxy install --roles-path . geerlingguy.apache
 
-   $ ansible-galaxy install -r requirements.txt
+.. seealso::
 
-Where the requirements.txt looks like::
+   :doc:`intro_configuration`
+      All about configuration files
```
Comment (bcoca) [inline docsite/rst/galaxy.rst:58]: This file is deprecated in favor of the YAML format and should be removed in 2.3

File: docsite/rst/galaxy.rst [hunk 3]
```diff
@@ -129,48 +128,100 @@ And here's an example showing some specific version downloads from multiple sour
       scm: git
       version: 0.1.0
 
-As you can see in the above, there are a large amount of controls available
-to customize where roles can be pulled from, and what to save roles as. 
+Installing multiple roles from multiple files
+=============================================
+
```
Comment (bcoca) [inline docsite/rst/galaxy.rst]: this was never implemented, i thought i removed these docs already

File: docsite/rst/galaxy.rst [hunk 4]
```diff
@@ -129,48 +128,100 @@ And here's an example showing some specific version downloads from multiple sour
...
+    - include: /etc/roles/web_server_roles.yml
+    - include: /var/roles/security.yml
+
+    # Additional roles
+    - src: geerlingguy.apache
+    - src: https://github.com/some_org/install_me.git
+      version: 1.2.0
+
+Muliple roles from a single repo 
+================================
```
Comment (bcoca) [inline docsite/rst/galaxy.rst]: do we really want to support this?

**Comment by bcoca** [comment]
We should also mention that the 'galaxy server' is configurable in case they want to host their own.

File: docsite/rst/galaxy.rst (hunk 1 above)
Comment (chouseknecht) [reply docsite/rst/galaxy.rst]: Good point. Updated.

File: docsite/rst/galaxy.rst (hunk 2 above)
Comment (chouseknecht) [reply docsite/rst/galaxy.rst:58]: Yes. References to _requirements.txt_ have been removed.

File: docsite/rst/galaxy.rst (hunk 4 above)
Comment (chouseknecht) [reply docsite/rst/galaxy.rst]: Nope. Removed.

File: docsite/rst/galaxy.rst (hunk 3 above)
Comment (chouseknecht) [reply docsite/rst/galaxy.rst]: It looks to me like it is implemented: https://github.com/ansible/ansible/blob/devel/lib/ansible/cli/galaxy.py#L358

I have not actually tested, so maybe it's not fully implemented?

Comment (chouseknecht) [reply docsite/rst/galaxy.rst]: Removed references to _includes_ for now. We can add it later, when/if it's fully formed and tested.

**Pull request merged**